zoos/zoos-router.js

```diff
@@ -0,0 +1,93 @@
+const router = require('express').Router();
+const knex = require('knex');
+
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+      filename: './data/lambda.sqlite3'
+    },
+  };
+  
+  const db = knex(knexConfig);
+
+router.get('/', (req, res) => {
+    db('zoos')
+    .then(zoos => {
+        res.status(200).json(zoos);
+    })
+    .catch(error => {
+        res.status(500).json(error);
+    })
+})
+
+router.get('/:id', (req, res) => {
+    const { id } = req.params;
+
+    db('zoos')
+    .where({ id })
+    .first()
+    .then(zoo => {
+        res.status(200).json(zoo);
+    })
+    .catch(error => {
+        res.status(500).json(error);
+    })
+})
+
+router.post('/', (req, res) => {
+    const { name } = req.body;
+
+    if(name !== undefined || name !== "") {
```

Comment:
```suggestion
    if(name) {
```
should produce the same result